From 1656292807d60c18a603a9c414677fa3a39b008f Mon Sep 17 00:00:00 2001
From: Bastian Blank <waldi@debian.org>
Date: Sat, 5 Jul 2014 11:46:45 +0200
Subject: config-prefix.diff

---
 Config.mk        | 2 +-
 config/StdGNU.mk | 9 ++++++---
 2 files changed, 7 insertions(+), 4 deletions(-)

diff --git a/Config.mk b/Config.mk
index f23b9d6..e7065e2 100644
--- a/Config.mk
+++ b/Config.mk
@@ -84,7 +84,7 @@ EXTRA_LIB += $(EXTRA_PREFIX)/lib
 endif
 
 PYTHON      ?= python
-PYTHON_PREFIX_ARG ?= --prefix="$(PREFIX)"
+PYTHON_PREFIX_ARG ?= --home="$(PRIVATE_PREFIX)"
 # The above requires that PREFIX contains *no spaces*. This variable is here
 # to permit the user to set PYTHON_PREFIX_ARG to '' to workaround this bug:
 #  https://bugs.launchpad.net/ubuntu/+bug/362570
diff --git a/config/StdGNU.mk b/config/StdGNU.mk
index 25fc594..b566b5c 100644
--- a/config/StdGNU.mk
+++ b/config/StdGNU.mk
@@ -30,16 +30,19 @@ INSTALL_PROG = $(INSTALL) -m0755 -p
 PREFIX ?= /usr
 BINDIR = $(PREFIX)/bin
 INCLUDEDIR = $(PREFIX)/include
-LIBEXEC = $(PREFIX)/lib/xen/bin
+LIBDIR = $(PREFIX)/lib
 SHAREDIR = $(PREFIX)/share
 MANDIR = $(SHAREDIR)/man
 MAN1DIR = $(MANDIR)/man1
 MAN8DIR = $(MANDIR)/man8
 SBINDIR = $(PREFIX)/sbin
-XENFIRMWAREDIR = $(PREFIX)/lib/xen/boot
 
-PRIVATE_PREFIX = $(LIBDIR)/xen
+PRIVATE_PREFIX = $(LIBDIR)/xen-$(XEN_VERSION)
 PRIVATE_BINDIR = $(PRIVATE_PREFIX)/bin
+PRIVATE_LIBDIR = $(PRIVATE_PREFIX)/lib
+
+LIBEXEC = $(PRIVATE_BINDIR)
+XENFIRMWAREDIR = $(PRIVATE_PREFIX)/boot
 
 CONFIG_DIR = /etc
 XEN_LOCK_DIR = /var/lock
