From edac38c6428c11764047b4c3847d7ee9d2d20e80 Mon Sep 17 00:00:00 2001
From: Bastian Blank <waldi@debian.org>
Date: Sat, 5 Jul 2014 11:46:56 +0200
Subject: tools-libfsimage-rpath.diff

---
 tools/libfsimage/Rules.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/libfsimage/Rules.mk b/tools/libfsimage/Rules.mk
index 9aedcc7..2750f4f 100644
--- a/tools/libfsimage/Rules.mk
+++ b/tools/libfsimage/Rules.mk
@@ -3,6 +3,7 @@ include $(XEN_ROOT)/tools/Rules.mk
 CFLAGS += -Wno-unknown-pragmas -I$(XEN_ROOT)/tools/libfsimage/common/ -DFSIMAGE_FSDIR=\"$(FSDIR)\"
 CFLAGS += -Werror -D_GNU_SOURCE
 LDFLAGS += -L../common/
+LDFLAGS += $(call LDFLAGS_RPATH,../..)
 
 PIC_OBJS := $(patsubst %.c,%.opic,$(LIB_SRCS-y))
 
