From 2586599a8d40916dcdba19d83d6ea64997c7045a Mon Sep 17 00:00:00 2001
From: Bastian Blank <waldi@debian.org>
Date: Sat, 5 Jul 2014 11:47:11 +0200
Subject: tools-xenstat-rpath.diff

---
 tools/xenstat/xentop/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tools/xenstat/xentop/Makefile b/tools/xenstat/xentop/Makefile
index 6146491..61f1c7d 100644
--- a/tools/xenstat/xentop/Makefile
+++ b/tools/xenstat/xentop/Makefile
@@ -19,7 +19,9 @@ all install xentop:
 else
 
 CFLAGS += -DGCC_PRINTF -Wall -Werror $(CFLAGS_libxenstat)
+LDFLAGS += $(call LDFLAGS_RPATH,../lib)
 LDLIBS += $(LDLIBS_libxenstat) $(CURSES_LIBS) $(SOCKET_LIBS)
+LDLIBS += $(LDLIBS_libxenctrl) $(LDLIBS_libxenstore)
 CFLAGS += -DHOST_$(XEN_OS)
 
 # Include configure output (config.h) to headers search path
