From 90380bf46fb8ed13ee7b896f215c645008f992e1 Mon Sep 17 00:00:00 2001
From: Bastian Blank <waldi@debian.org>
Date: Sat, 5 Jul 2014 11:47:13 +0200
Subject: tools-xenstore-rpath.diff

---
 tools/xenstore/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tools/xenstore/Makefile b/tools/xenstore/Makefile
index f8237ed..153c36f 100644
--- a/tools/xenstore/Makefile
+++ b/tools/xenstore/Makefile
@@ -8,6 +8,8 @@ CFLAGS += -Werror
 CFLAGS += -I.
 CFLAGS += $(CFLAGS_libxenctrl)
 
+LDFLAGS_libxenctrl += $(call LDFLAGS_RPATH,../lib)
+
 CLIENTS := xenstore-exists xenstore-list xenstore-read xenstore-rm xenstore-chmod
 CLIENTS += xenstore-write xenstore-ls xenstore-watch
 
