From f0078305a097d55a4028d44054b3e1edc9d8f0b0 Mon Sep 17 00:00:00 2001
From: Bastian Blank <waldi@debian.org>
Date: Sat, 5 Jul 2014 11:47:15 +0200
Subject: tools-xentrace-rpath.diff

---
 tools/xentrace/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/xentrace/Makefile b/tools/xentrace/Makefile
index a559110..fcc578a 100644
--- a/tools/xentrace/Makefile
+++ b/tools/xentrace/Makefile
@@ -5,6 +5,7 @@ CFLAGS += -Werror
 
 CFLAGS += $(CFLAGS_libxenctrl)
 LDLIBS += $(LDLIBS_libxenctrl)
+LDFLAGS += $(call LDFLAGS_RPATH,../lib)
 
 BIN      = xentrace xentrace_setsize
 LIBBIN   = xenctx
